openmeetings-web/src/main/java/org/apache/openmeetings/web/room/wb/WbPanel.html

```diff
@@ -39,10 +39,14 @@
 		<div id="wb-tabbar-ctrls-right">
 			<div class="next clickable om-icon big"></div>
 		</div>
-		<ul><li id="wb-area-tab" class="nav-item">
-			<a class="nav-link" data-toggle="tab" role="tab"><span></span></a>
-		</li></ul>
-		<button id="wb-tab-close" class="btn btn-sm btn-outline-secondary"
```

Comment:
removing `btn-outline-secondary` also removes `hover` effect

So I'll leave it to you
I do like how this button looks now, but I miss previous `hover` effect

Reply:
I fixed it so those icon-buttons on the whiteboard menu have a hover. But the border is gone.
![image](https://user-images.githubusercontent.com/5152064/79079741-f17e2880-7d64-11ea-9cef-4e742bad6e74.png)

I just don't understand why buttons on Bootstrap have no icon mode. Its clearly that all those button designs are meant for buttons with text:
https://getbootstrap.com/docs/4.0/components/buttons/

But those are icon-buttons. There is no text. I can understand for text to have border, cause otherwise you won't recognise it clickable. But for an icon, I don't quite agree this would require a border radius around it all the time.

Reply:
I don't really like how its spacing in between those menu items. Also it would look much better if the zoom is on the left (as its always visible). Cause at the moment it seems to free-flowing in the middle (while wonder what that empty whitespace on the left is good for - which is only visible once there is a document on the whiteboard - for paging it).

I don't quite get what the spacing is good for either.

Reply:
Dialog close button have no text and have border in `jquery-ui`
I'm OK with some buttons has np border

But IMO zoom control looks more consistent with icons with borders
@dkandrov what do you think?

Reply:
I agree with @sebawagner about `X` button on board tabs. They look better without borders.

About paging and zoom groups I think it will be better to leave borders because they are groups of buttons which related to the same functionality. Maybe it will be better to show paging all time but in a disabled state when it is not necessary. And move all controls to one side without stretching them on all wide.
Something like below:
![изображение](https://user-images.githubusercontent.com/5506989/79197867-b95a1100-7e5c-11ea-9a01-d59a0868aa03.png)

Reply:
OK @dkandrov sounds good. I reverted the change on the other buttons and kept only the close icon without border:
![image](https://user-images.githubusercontent.com/5152064/79297465-55098100-7f32-11ea-9365-c7648a330683.png)

in the commit 89a006c

And for the modifications on the other controls on the whiteboard we can use another Jira/PR